packages/common/src/schema.graphql

```diff
@@ -318,31 +341,44 @@ type InsertOutboundShipmentResponseWithId {
   response: InsertOutboundShipmentResponse!
 }
 
+type InternalError implements UserRegisterErrorInterface & AuthTokenErrorInterface & LogoutErrorInterface & RefreshTokenErrorInterface {
+  description: String!
+  fullError: String!
+}
+
+type InvalidCredentials implements AuthTokenErrorInterface {
+  description: String!
+}
+
+type InvalidToken implements RefreshTokenErrorInterface & LogoutErrorInterface {
+  description: String!
+}
+
 # Generic Connector
 type InvoiceConnector {
   totalCount: Int!
   nodes: [InvoiceNode!]!
 }
 
-type InvoiceDoesNotBelongToCurrentStore implements DeleteOutboundShipmentErrorInterface & DeleteInboundShipmentLineErrorInterface & UpdateInboundShipmentErrorInterface & DeleteOutboundShipmentLineErrorInterface & UpdateOutboundShipmentLineErrorInterface & DeleteInboundShipmentErrorInterface & UpdateInboundShipmentLineErrorInterface & InsertOutboundShipmentLineErrorInterface & InsertInboundShipmentLineErrorInterface {
+type InvoiceDoesNotBelongToCurrentStore implements InsertOutboundShipmentLineErrorInterface & DeleteInboundShipmentLineErrorInterface & InsertInboundShipmentLineErrorInterface & UpdateInboundShipmentErrorInterface & UpdateOutboundShipmentLineErrorInterface & UpdateInboundShipmentLineErrorInterface & DeleteOutboundShipmentErrorInterface & DeleteOutboundShipmentLineErrorInterface & DeleteInboundShipmentErrorInterface {
   description: String!
 }
 
 input InvoiceFilterInput {
-  ## Additions
   otherPartyName: SimpleStringFilterInput
-  nameId: SimpleStringFilterInput
-  storeId: SimpleStringFilterInput
-  type: SimpleStringFilterInput
-  status: SimpleStringFilterInput
+  invoiceNumber: EqualFilterNumberInput
+  nameId: EqualFilterStringInput
+  storeId: EqualFilterStringInput
+  type: EqualFilterInvoiceTypeInput
+  status: EqualFilterInvoiceStatusInput
   comment: SimpleStringFilterInput
-  theirReference: SimpleStringFilterInput
+  theirReference: EqualFilterStringInput
```

Comment:
I reverted this change - don't really need to change it, but I do think it's weird